### ISSUE TITLE
The order of predicted value

### ISSUE THREAD

**Issue by naush01** [ultralytics/yolov5#11844]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello, 

The code below is extracted from ```class Detect``` in ```yolo.py```
```
    def forward(self, x):
        z = []  # inference output
        for i in range(self.nl):
            x[i] = self.m[i](x[i])  # conv
            bs, _, ny, nx = x[i].shape  # x(bs,255,20,20) to x(bs,3,20,20,85)
            x[i] = x[i].view(bs, self.na, self.no, ny, nx).permute(0, 1, 3, 4, 2).contiguous()

            if not self.training:  # inference
                if self.dynamic or self.grid[i].shape[2:4] != x[i].shape[2:4]:
                    self.grid[i], self.anchor_grid[i] = self._make_grid(nx, ny, i)

                if isinstance(self, Segment):  # (boxes + masks)
                    xy, wh, conf, mask = x[i].split((2, 2, self.nc + 1, self.no - self.nc - 5), 4)
                    xy = (xy.sigmoid() * 2 + self.grid[i]) * self.stride[i]  # xy
                    wh = (wh.sigmoid() * 2) ** 2 * self.anchor_grid[i]  # wh
                    y = torch.cat((xy, wh, conf.sigmoid(), mask), 4)
                else:  # Detect (boxes only)
                    xy, wh, conf = x[i].sigmoid().split((2, 2, self.nc + 1), 4)
                    xy = (xy * 2 + self.grid[i]) * self.stride[i]  # xy
                    wh = (wh * 2) ** 2 * self.anchor_grid[i]  # wh
                    y = torch.cat((xy, wh, conf), 4)
                z.append(y.view(bs, self.na * nx * ny, self.no))
        return x if self.training else (torch.cat(z, 1), ) if self.export else (torch.cat(z, 1), x)
```

My question is how can we know the order of ```x[i].sigmoid().split((2, 2, self.nc + 1), 4)``` is ```xy, wh, conf```
```
xy, wh, conf = x[i].sigmoid().split((2, 2, self.nc + 1), 4)
```

Thank you!

**Comment by glenn-jocher** [comment]
@naush01 hi,

In the code snippet you provided, the order of the returned values from `x[i].sigmoid().split((2, 2, self.nc + 1), 4)` is `xy`, `wh`, and `conf`.

The `split()` function in PyTorch splits a tensor into multiple tensors along a specified axis. In this case, the tensor `x[i].sigmoid()` is split into four tensors using the values `(2, 2, self.nc + 1)` as the split sizes along the fourth dimension (axis 4). The resulting tensors are `xy`, `wh`, `conf`, and an additional tensor.

Please note that understanding the order of the returned values from a particular code segment may require examining the context and the purpose of the code. If you have any further questions or need clarification, feel free to ask.

Thank you for your interest in YOLOv5!
